cheatsheets/Cross_Site_Scripting_Prevention_Cheat_Sheet.md

```diff
@@ -1,400 +1,207 @@
-# Cross Site Scripting Prevention Cheat Sheet
+# Cross Site Scripting (XSS) Prevention Cheat Sheet
```

Comment:
AFAIK having the shorter version in the title isn't a great practice, and having it defined in the first sentence is better.
Makes linking as well easier on processors (sometimes they break on special chars, `(` and `)` in this case)

Reply:
I agree with @ThunderSon

Reply:
Accepted.